.vscode/launch.json

```diff
@@ -7,13 +7,14 @@
         {
             "type": "node",
             "request": "launch",
-            "name": "Unit tests",
+            "name": "Unit tests - OETH Swapper",
```

Comment:
Is this the correct name? Sounds like it's just running OETH Swapper tests.

Reply:
The name is correct as it has the `test/vault/oeth-swapper.js` param. I'll add another launcher for all unit tests